src/main/java/com/github/bvale/helloSpring/controller/TodoController.java

```diff
@@ -28,7 +25,7 @@
 
 
     @ResponseBody
-    @RequestMapping(value = "/list", method = GET)
+    @RequestMapping(method = GET)
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" ([squid:S4488](https://sonarcloud.io/organizations/bernardovale-github/rules#rule_key=squid%3AS4488))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.github.bvale%3AhelloSpring&pullRequest=1&issues=AWSuJ7bd6dA5b-W2m-sd&open=AWSuJ7bd6dA5b-W2m-sd)

[ISSUE_KEY]: AWSuJ7bd6dA5b-W2m-sd